books/RayTracingInOneWeekend.html

```diff
@@ -501,6 +501,25 @@
 I will have the y-axis go up, and the x-axis to the right.
 In order to respect the convention of a right handed coordinate system, into the screen is the
 negative z-axis.
+
+Now the inevitable tricky part.
+While our 3D space has the conventions above, this conflicts with the standard _image_ coordinates,
+where the Y axis is inverted.
+In image coordinates, Y=0 denotes the top of the image, and Y=1 denotes the bottom.
+
+(This is very handy if your pixels are read out from a scanline buffer to feed an electron beam that
+scans top to bottom, left to right, exciting screen phosphors as it goes.
+Your display doesn't work like that?
```

Comment:
I think a ton of our audience is going to mistake this joke.

Consider rewriting for obviousness, or remove

Reply:
Yeah, I'll drop.